sochain/test/adapter.test.ts

```diff
@@ -0,0 +1,167 @@
+import { assert } from 'chai'
```

Comment:
Please think about how should we reuse this as a spec that gets called by specific tests in a specific adapter. I assume this is just C/P for all bitcoin indexer adapters.